### DISCUSSION TITLE
cannot use the models

### DISCUSSION THREAD

**Discussion by speedhawk** [explosion/spaCy#10381] [Help: Installation]
After I install the scispacy, it cannot use spacy.load("en_core_sci_sm") function with an error called " Could not read config file from C:\Users\Lenovo\AppData\Roaming\Python\Python39\site-packages\en_core_sci_sm\en_core_sci_sm-0.2.5\config.cfg".

**Comment by koaning** [comment]
Just to confirm, could you list all the steps you've taken beforehand? Did you run the code below? 

```
python -m pip install scispacy
python -m pip install en_core_sci_sm
```

**Reply by speedhawk** [reply]
Thank you for your reply. Actually, I cannot execute the first introduction with lots of errors of 'nmslib’ library installation but the second one can be correctly operated. 
Detaily, the errors appears as "legacy-install-failure" whenever I use pip or directly "setup.py".

**Reply by koaning** [reply]
Try running this first:

```
python -m pip install --upgrade pip
```

**Reply by speedhawk** [reply]
> 

Actually, it reminded me that the version of pip is not the reason of installation failure but the subprocess is. I try it as you said and it still not works.

**Reply by koaning** [reply]
Could you share the full traceback?
